docker-compose.yml

```diff
@@ -17,9 +17,10 @@ services:
     ports:
     - 8000:80
   netbox:
-    image: "vaporio/netbox:develop.5"
+    image: "vaporio/netbox:develop"
     environment:
       SECRET_KEY: secret
+      SUPERUSER_API_TOKEN: 2307e5ae8b3d-CI-CODED-c67cc44105c36
```

Comment:
ah this is an artifact. do we want to keep it or ditch it? i use this when testing the ci-pipeline flow... its maybe not necessary since it defaults to something even worse than this.